Cargo.toml

```diff
@@ -27,7 +27,9 @@ glob = "0.3"
 humantime = "2"
 itertools = "0.10"
 log = "0.4"
+postgres-types = { version = "0.2.3", features = ["derive","with-chrono-0_4"] }
```

Comment:
please make this consistent with other lines :) Also I think it would be better to only enable Postgres-types in bin target?